test

```diff
@@ -185,16 +188,12 @@ def test_all(year, event, problem, language, id):
 
   checker = prepare_checker(language, file)
 
-  results = [
-    test(checker, input, expected)
-    for input, expected in test_case
-  ]
+  for input, expected in test_case:
+    yield test(checker, input, expected)
```

Comment:
Maybe I didn't something the pythonic way but. I don't think it's inside a `class` scope. Am I wrong?